R/update-news.R

```diff
@@ -27,14 +27,16 @@ update_news_impl <- function(commits, which) {
       which <- "dev"
     }
   }
+  no_news.md_yet <- is.null(fledgeling[["news"]])
```

Comment:
🙀 

```suggestion
  initializing <- is.null(fledgeling[["news"]])
```

Other candidates: `just_hatching` or `brand_new`, but this would be confusing too.